packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
+* **BREAKING CHANGE**: Removes `launchPriceChangeConfirmationFlow` from `InAppPurchaseAndroidPlatform` as it is deprecated by Android.
```

Comment:
Generally we like to provide migration tips with breaking changes; is there a replacement we can point people to?

Reply:
I will update the CHANGELOG mentioning that price flow changes are now handled by Google Play. Can I link developers to a relevant url or do we stray away from using those?

Reply:
A link to relevant docs sounds great!